docs/source/configure.rst

```diff
@@ -12,6 +12,13 @@ When you run the command to start the Quipucords server, you supply values for s
 - Selecting a directory for SSH keys
 - Selecting a directory for the SQLlite database
 - Selecting a directory for log output
+- Selecting a database (postgres or sqlite3) by setting `QPC_DBMS`
+- If you are using postgres as your database, you will also have the following options:
```

Comment:
Should we indent these so they are underneath `Selecting a database (postgres or sqlite3) by setting QPC_DBMS`